libr/core/cmd_anal.c

```diff
@@ -6207,6 +6208,35 @@ static bool cmd_anal_refs(RCore *core, const char *input) {
 		free (tInput);
 		free (fInput);
 	} break;
+	case 'm': { // "axm"
+		RList *list = NULL;
+		RAnalRef *ref;
+		RListIter *iter;
+		char *ptr = strdup (r_str_trim_head ((char *)input + 1));
+		int n = r_str_word_set0 (ptr);
+		ut64 at = core->offset;
+		ut64 addr = UT64_MAX;
+		switch (n) {
+		case 2: // get at
+			at = r_num_math (core->num, r_str_word_get0 (ptr, 1));
+		/* fall through */
+		case 1: // get addr
+			addr = r_num_math (core->num, r_str_word_get0 (ptr, 0));
+			break;
+		default:
+			free (ptr);
+			return false;
+		}
+		//get all xrefs pointing to addr
+		list = r_anal_xrefs_get (core->anal, addr);
+		r_list_foreach (list, iter, ref) {
+			r_cons_printf ("0x%p %s\n", ref->addr, r_anal_xrefs_type_tostring(ref->type));
```

Comment:
mnissing space before (

Reply:
use PFMT64X instead pf %p, this is not portable

Reply:
we can probably p`rint the r2 commands to achieve the same instead of printing and calling the api, or at least when axm* is used

Reply:
> we can probably p`rint the r2 commands to achieve the same instead of printing and calling the api, or at least when axm* is used

Not entirely sure what you mean by this. Can you give an example where this is done elsewhere?